src/.editorconfig

```diff
@@ -1,5 +1,14 @@
 root = true
 
+# Don't use tabs for indentation.
+[*]
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
I highly recommend _not_ making this change. The handling for this property does not consider the semantics of languages involved, and may silently introduce behavior changes in the future.

Reply:
Which line specifically? Or are you talking about all of them? Listing every text file extension we use that I'd want this to apply is possible, but would be a pain. I'd lean toward leaving it like this until it becomes a problem, since AFAIK I don't ever use, edit or check in *any* text file format to git that would malfunction in any of these cases.
(I don't work with Python)

Reply:
I'm referring only to line 7.

Reply:
Thanks. Can you tell me from your experience what the most likely problem is that would bite me if I left this in?

Reply:
It may never be a problem. One case where it could matter is if a test is somehow relying on this, and the test changes behavior (without failing) as a result of the spacing change, leading to a test gap and possibility of a future false-negative. I've also seen some cases where tools add these spaces, and then other tools try to follow the configuration and remove it, leading to unnecessary churn in the changes. The setting does not account for "generated" files.